nilearn/image/image.py

```diff
@@ -902,12 +908,22 @@ def clean_img(imgs, sessions=None, detrend=True, standardize=True,
                 "specified None. imgs header suggest it to be {0}".format(
                     imgs.header.get_zooms()[3]))
 
-    data = signal.clean(
-        imgs_.get_data().reshape(-1, imgs_.shape[-1]).T, sessions=sessions,
-        detrend=detrend, standardize=standardize, confounds=confounds,
-        low_pass=low_pass, high_pass=high_pass, t_r=t_r,
-        ensure_finite=ensure_finite).T.reshape(imgs_.shape)
-    return new_img_like(imgs, data, copy_header=True)
+    # Store function parameters in dictionary for easy access
+    clean_paramters = {
```

Comment:
I think I'd clean the parameters explicit in the two calls of the function.

Reply:
Not sure if I understand you correctly. You mean, I should have them twice written out? As they are exactly the same, I thought to prevent redundancy.

Reply:
> Not sure if I understand you correctly. You mean, I should have them twice written out? As they are exactly the same, I thought to prevent redundancy.

Sorry I'd just pass all the arguments explictily in the calls of the function. makes it easier to inspect tracebacks etc.